docker/analysis-tools/Dockerfile

```diff
@@ -68,14 +71,17 @@ RUN Rscript -e "install.packages('ggplot2')"
 RUN Rscript -e "install.packages('googleCloudStorageR')"
 
 # Mount Google Cloud secret key
-ADD broad-dsde-mint-dev-ed8cc09c48e5.json /usr/secrets/broad-dsde-mint-dev-ed8cc09c48e5.json
+ADD broad-dsde-mint-dev-testR.json /usr/secrets/broad-dsde-mint-dev-testR.json
```

Comment:
are they the same json file as before but renamed?

Reply:
Yeah, they are the same, just renamed it to a more human-readable name.